cluster/spanning_tree_main.cc

```diff
@@ -17,33 +17,70 @@ int getpid() { return (int)::GetCurrentProcessId(); }
 
 #include <iostream>
 #include <fstream>
-#include <string.h>
-#include <stdlib.h>
-#include <errno.h>
 
 using namespace VW;
 
+#include <boost/program_options.hpp>
+namespace po = boost::program_options;
+
+void usage(const po::options_description& desc)
+{
+  std::cout << "usage: spanning_tree [--port,-p number] [--nondaemon] [--help,-h] [pid_file]" << std::endl;
```

Comment:
is the boost print help message easy to incorporate?

Reply:
Thats what line 29 is. It doesn't give you one in this form though and it is important to signal the final positional arg